src/facade/cmd_arg_parser.h

```diff
@@ -90,6 +90,11 @@ struct CmdArgParser {
     }
   }
 
+  // returns next value if exists or default value
+  template <class T = std::string_view, class... Ts> auto NextOrDefault(T default_value = {}) {
+    return HasNext() ? Next<T, Ts...>() : default_value;
+  }
+
```

Comment:
No need for the variadic, it will have a malformed return type: `bool ? tuple<> : T`,  i.e. NextOrDefault<int,int,int>